src/App.js

```diff
@@ -1,36 +1,25 @@
 import React from 'react';
-import { Router, Link } from '@reach/router';
-
+import { Router } from '@reach/router';
 import Home from './Home/components/Home';
 import MakeListingDisplay from './MakeListingDisplay/components/MakeListingDisplay';
-
 import './App.css';
+import SignInPage from "./AuthenticationApp/MainComponents/SignInPage";
+import SignUpPage from "./AuthenticationApp/MainComponents/SignUpPage";
+import UserWrappedMain from "./AuthenticationApp/MainComponents/UserWrappedMain";
+import PostCommentsDisplay from './PostCommentsDisplay/components/PostCommentsDisplay';
 
 /** Component class to render the whole comments page */
-function App(props) {
+function App () {
     return (
-        <div>
-            <h1>Roomies</h1>
-
-            <nav>
-                <Link to="/">
-                    <button type="button">
-                        Home
-                    </button>
-                </Link>
-                <Link to="/makeListing">
-                    <button type="button">
-                        Make a Listing
-                    </button>
-                </Link>
-            </nav>
-
-            <Router>
-                <Home path="/" />
-                <MakeListingDisplay path="/makeListing" />
-            </Router>
-        </div>
```

Comment:
I think you're changing an old version of our project. If you can, try to pull the changes made and update this pr

Reply:
Didn't realize I hadn't pulled from master in a while, went back and merged and added a new commit